dispatch/static/manager/src/js/pages/EventAudit/index.js

```diff
@@ -0,0 +1,181 @@
+  limit: PER_PAGE
+}
+
+class EventAuditPage extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      count: 0
+    }
+  }
+
+  getQuery(props) {
+    props = props || this.props
+
+    const query = R.clone(PENDING_QUERY)
+
+    if (props.location.query.page) {
+      query.offset = (Number(props.location.query.page) - 1) * PER_PAGE
+    }
+
+    return query
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.events.count != this.props.events.count) {
+      this.setState({ count: nextProps.events.count })
+    }
+
+    if (nextProps.events.count && !this.eventCount
+        && !this.reloadListFlag) {
+      this.props.listEvents(this.props.token, this.getQuery(nextProps))
+      this.reloadListFlag = true
+    } else if (this.props.location.query.page != nextProps.location.query.page){
+      this.props.listEvents(this.props.token, this.getQuery(nextProps))
+    }
+
+  }
+
+  componentWillMount() {
+    this.props.listEvents(this.props.token, this.getQuery())
+
+    if (this.props.events) {
+      this.setState({ count: this.props.events.count })
+    }
+  }
+
+  checkPage() {
+    if (this.eventCount == 1 && this.props.location.query.page > 1) {
+      let query = R.clone(this.props.location.query)
+
+      query.page--
+
+      this.props.replace({
+        pathname: this.props.location.pathname,
+        query: query
+      })
+    }
+  }
+
+  approve(id) {
+    const event = this.props.entities.events[id]
+    event.is_submission = 0
+    this.props.saveEvent(this.props.token, id, event)
+    this.checkPage()
+    this.setState({ count: this.state.count - 1 })
+    this.reloadListFlag = false
+  }
+
+  disapprove(id) {
+    this.props.deleteEvent(this.props.token, id)
+    this.checkPage()
+    this.setState({ count: this.state.count - 1 })
+    this.reloadListFlag = false
+  }
+
+  render() {
+    this.eventCount = 0
+    const events = this.props.events.ids.map((id, idx) => {
+      const event = this.props.entities.events[id]
+
+      if (!event.is_submission) {
+        return null
+      }
+      this.eventCount++
+
+      return (
+        <EventCard
+          key={idx}
+          event={event}
+          approve={() => {this.approve(id, idx)}}
+          disapprove={() => {this.disapprove(id, idx)}} />
+      )
+    })
+
+    return (
+      <DocumentTitle title='Audit Event'>
+        <div className='u-container'>
+          <h2 className='c-event-audit-title'>
+            Event Auditing
+          </h2>
+          <div className='c-event-audit-subtitle'>
+            {this.state.count > PER_PAGE ?
+              <EventAuditPagination
+                page={Number(this.props.location.query.page || 1)}
+                pages={Math.ceil(this.state.count / PER_PAGE)}
+                location={this.props.location} />: null}
+            {this.state.count} event{this.state.count == 1 ? '' : 's'} pending approval
+          </div>
+          <ReactCSSTransitionGroup
```

Comment:
This transitions are nice, but I think it'd prefer to remove them for now since we don't use them anywhere else in the application